src/puppet-web/puppet-web.ts

```diff
@@ -455,17 +468,22 @@ export class PuppetWeb extends Puppet {
       destinationId = to.id
     }
 
-    const mediaId = await this.uploadMedia(message, destinationId)
+    const { mediaId, mediaData } = await this.uploadMedia(message, destinationId)
     const msgType = UtilLib.msgType(message.ext())
 
+    mediaData.FileType = 7
```

Comment:
After testing, no valid use of the file type parameter has been found and has been removed.